test/cpp/api/operations.cpp

```diff
@@ -55,3 +55,33 @@ TEST_F(OperationTest, Cross) {
     ASSERT_TRUE(out.allclose(exp));
   }
 }
+
+TEST_F(OperationTest, Linear_out) {
```

Comment:
I remember Nick posted some comments about this test, mainly on the integer type. Can we use an approach similar to `TEST_F(OperationTest, Cross)` where we use `torch::rand` to generate the input and use loops to compute the expected result?

Reply:
@desertfire nick added a comment about using `OpInfo` instead of this approach. I agree it's a better approach but I would like to land this one and then submit the `OpInfo` in a separate PR.